text-ui-test/EXPECTED.TXT

```diff
@@ -1,7 +1,136 @@
-Hello from
- ____        _        
-|  _ \ _   _| | _____ 
-| | | | | | | |/ / _ \
-| |_| | |_| |   <  __/
-|____/ \__,_|_|\_\___|
+____________________________________________________________
+ Hello! I'm Duke
+ What can I do for you?
+____________________________________________________________
 
+>> ____________________________________________________________
+ Gotcha! I've added this task: 
+	[T][?] read book
```

Comment:
Should this either be a cross/tick?

Reply:
This is in the EXPECTED.txt file, not really a coding standard/quality violation :), it just doesn't display properly since its in txt.